### DISCUSSION TITLE
Broken link (No more eagle files)

### DISCUSSION THREAD

**Discussion by CaitSith2** [sanni/cartreader#281] [General Chat]
https://github.com/sanni/cartreader/blob/993286f51b8be7a1fa631797bb7c406037a42382/pcb/adapters/README.md?plain=1#L5

When you did the spring cleaning and deleted all of the eagle source files, this link became broken.

Kind of curious as to if there are plans to provide kicad sources for all of the adapter boards.

**Comment by sanni** [comment]
Yes that's the plan. Currently I'm waiting for KiCad 6 to release.

**Comment by LeSuedois** [comment]
Hello @sanni , really thank you for your work on this project.
Please tell me, I am looking for eagle libraries too. I just don't feel ready to switch to KiCad since I use Eagle since ~25years..
Is it possible to share !sanni_mod3.lbr library somewhere? 
I want to do some Raspberry Pi Pico Gameboy dumper and A little tired spending my weekend doing a footprint.

**Comment by sanni** [comment]
I'm sorry I deleted all the Eagle files and moved on to KiCad. I don't like what Eagle PCB has become and I truly hope that all the makers will leave it behind and KiCad will become the new standard. 
I wish you the best with your project.
